Engine/source/gfx/gl/tGL/tGL.h

```diff
@@ -24,7 +24,8 @@
 #define T_GL_H
 #include "GL/glew.h"
 
-#define gglHasExtension(EXTENSION) GLEW_##EXTENSION
+// Slower but reliably detects extensions
+#define gglHasExtension(EXTENSION) glewGetExtension("GL_" # EXTENSION)
```

Comment:
I think this is because we create GL contexts with different parameters in the win32-native bits of the GL driver. I don't have the hardware to debug this myself though. I can see if I can debug this on a VM maybe. Does Windows come with a software GL renderer that supports GL3+?